docs/en/writing-running-appium/caps.md

```diff
@@ -55,6 +55,7 @@ These Capabilities span multiple drivers.
 |`eventTimings`|Enable or disable the reporting of the timings for various Appium-internal events (e.g., the start and end of each command, etc.). Defaults to `false`. To enable, use `true`. The timings are then reported as `events` property on response to querying the current session. See the [event timing docs](/docs/en/advanced-concepts/event-timings.md) for the the structure of this response.|e.g., `true`|
 |`enablePerformanceLogging`| (Web and webview only) Enable Chromedriver's (on Android) or Safari's (on iOS) performance logging (default `false`)| `true`, `false`|
 |`printPageSourceOnFindFailure`| When a find operation fails, print the current page source. Defaults to `false`.|e.g., `true`|
+|`clearSystemFiles`| Delete any generated files at the end of a session. Default to `false`. | `true`, `false`|
```

Comment:
Android case only deletes the temporary app file as `app` capability. Mostly for iOS.